subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/transport/ExternalStorageTransport.kt

```diff
@@ -19,25 +19,30 @@ class ExternalStorageTransport(
     private val logger = loggerFactory.create<ExternalStorageTransport>()
 
     override fun send(state: ReportState.Initialized.Started) {
-        val testFolderName = "${state.testMetadata.className}#${state.testMetadata.methodName}"
+        try {
+            val testFolderName = "${state.testMetadata.className}#${state.testMetadata.methodName}"
 
-        val testMetadataDirectory = testMetadataDirectory(testFolderName = testFolderName)
+            val testMetadataDirectory = testMetadataDirectory(testFolderName = testFolderName)
 
-        val reportMetadataJson = File(testMetadataDirectory, REPORT_FILE_NAME)
+            val reportMetadataJson = File(testMetadataDirectory, REPORT_FILE_NAME)
 
-        val testRuntimeDataPackage = TestRuntimeDataPackage(
-            incident = state.incident,
-            dataSetData = state.dataSet?.serialize() ?: emptyMap(),
-            video = state.video,
-            preconditions = transformStepList(state.preconditionStepList),
-            steps = transformStepList(state.testCaseStepList),
-            startTime = state.startTime,
-            endTime = state.endTime
-        )
+            val testRuntimeDataPackage = TestRuntimeDataPackage(
+                incident = state.incident,
+                dataSetData = state.dataSet?.serialize() ?: emptyMap(),
+                video = state.video,
+                preconditions = transformStepList(state.preconditionStepList),
+                steps = transformStepList(state.testCaseStepList),
+                startTime = state.startTime,
+                endTime = state.endTime
+            )
 
-        logger.debug("Write report for test $testFolderName to file: ${reportMetadataJson.absolutePath}")
+            logger.debug("Write report for test $testFolderName to file: ${reportMetadataJson.absolutePath}")
 
-        reportMetadataJson.writeText(gson.toJson(testRuntimeDataPackage))
+            reportMetadataJson.writeText(gson.toJson(testRuntimeDataPackage))
+        } catch (e: Throwable) {
+            // TODO LOST
```

Comment:
should write some "stub" report here with more info, for now warn log will be enough to find a reason

Reply:
It leads to test failure I think it's a `critical`